registry/mainnet/interchain/squid.tokenlist.json

```
@@ -6,47 +6,55 @@
     "minor": 1,
     "patch": 0
   },
-  "tokens": [
-    {
+  "tokens": {
+    "0xb689de707e78469c6a115fdcfd9a0c2db9f8e7ce5ade7b29526d63f4e01d77fb": {
       "tokenId": "0xb689de707e78469c6a115fdcfd9a0c2db9f8e7ce5ade7b29526d63f4e01d77fb",
       "tokenAddress": "0x524d524B4c9366be706D3A90dcf70076ca037aE3",
-      "tokenManagerAddress": "0x5F53895484af488e27fd33CE70a3dbD745aeAA52",
-      "tokenManagerType": "mint_burn",
+      "tokenManager": "0x5F53895484af488e27fd33CE70a3dbD745aeAA52",
+      "tokenManagerType": "mintBurn",
       "symbol": "RMRK",
       "prettySymbol": "RMRK",
       "decimals": 18,
       "name": "RMRK",
       "originAxelarChainId": "moonbeam",
-      "transferType": "interchain",
+      "tokenType": "interchain",
```

```suggestion
      "tokenType": "customInterchain",
```
maybe add the kind here as well?